index.js

```diff
@@ -0,0 +1,84 @@
+const db = require("./databaseFunctions.js");
+
+// TODO: Setup remote database connection instead of just locally hosted
+// TODO: check for internet connection to be able to grab items from database
+
+async function listTasks() {
+  await db
+    .getTasks()
+    .then((res) => {
+      if (res.length == 0) {
+        console.log("No tasks were found in database.");
+      } else {
+        res.forEach((task) => {
+          console.log(
+            `ID: ${task.id}, Task name: ${task.name}, Completion status: ${task.completed}`
+          );
+        });
+      }
+    })
+    .catch((err) => {
+      console.error("Error with tasks returned: " + err);
+    });
+}
+
+async function createTask(taskName) {
+  if (!taskName) {
+    console.error("Please enter a task name to create.");
+  }
+  if (typeof taskName !== "string") {
+    console.error("Please give the name of the task as a string.");
+  } else {
+    await db
+      .addTask(taskName)
+      .then((res) => {
+        console.log(`Task added with the ID of: ${res.id}`);
+      })
+      .catch((err) => {
+        console.error("Error with adding a task: " + err);
+      });
+  }
+}
+
+async function completeTask(taskID) {
+  if (!taskID) {
+    console.error("Please enter a task to be marked as complete.");
```

Comment:
We should probably specify a task ID here instead of just saying a task
```suggestion
    console.error("Please enter the ID of a task to be marked as complete.");
```